fields/types/localfile/LocalFileType.js

```diff
@@ -89,7 +89,7 @@ localfile.prototype.addToSchema = function() {
 	
 	var schemaPaths = this._path.addTo({}, {
 		filename:		String,
-		originalname :		String,
+		originalname:	String,
```

Comment:
we should use spaces for mid-line indentation, this will not line up with tab width is 4